### ISSUE TITLE
Community Check-in #4 Summer Of Code Week 5

### ISSUE THREAD

**Issue by ryzokuken** [publiclab/plots2#2814]
Hi everybody 👷👷‍♀️  ! 
If you're new here, welcome, and please checkout our [welcome page](https://publiclab.github.io/community-toolbox/#r=all)!! 👍 🎉 😄 

@stefannibrasil @milaaraujo @MargaretAN9 @Gauravano @namangupta01 @ViditChitkara @tech4GT @mridulnagpal @SidharthBansal @sagarpreet-chadha @Souravirus @MayankKashyap @siaw23 @ryzokuken @ccpandhare @icarito @abTest-123 @steviepubliclab @ebarry @publiclab/soc @publiclab/reviewers 

_How are you guys doing ? How was your weekend 🔆 ?_

**Leave a comment here describing what are your plans for this week ❓ or simply leave a  Hi if not planned yet 😄  !**

I have been a bit busy off the project (sorry for that), working with the awesome peeps at Node to make sure the project stays as awesome as possible at the Node.js collaborator summit in Berlin, Germany

![node summit](https://pbs.twimg.com/media/DenpAfWWkAA1a3t.jpg:large)

After that, I ended up in JSConf EU, which was pretty awesome as well.

P.S. We use Node in a ton of our own projects, including image-sequencer and plotsbot.

Meanwhile, @jywarren has been hard at work. So hard, in fact, that he was:

> I'm that guy compiling phusion passenger on the red line

Sneak peek:

![jeff's computer](https://matrix.org/_matrix/media/v1/download/matrix.org/NXERIwsOdZwSZkMDuwqGqSnO)

****

## 'Til next week!

As always, if you're waiting for a review, or if you're stuck,  please request help
here OR leave a comment with @publiclab/soc and @publiclab/reviewers for
some input.

Happy coding 🎉  !

**Comment by MayankKashyap** [comment]
Hi @ryzokuken it's good to see you back., And best of luck to every folk for their 5th week...and yes ujwal you are right @jwarren is really very hardworking..

**Comment by SidharthBansal** [comment]
Hi @publiclab/soc, @publiclab/reviewers 
I am great here. Almost done with basic sign up and log in with google in https://github.com/publiclab/plots2/pull/2771. Can someone please review it?

I have also set up facebook and github app settings and they are working on test app. I will be writing tests for google this week.
Once #2771 is merged. I will write the views and test for facebook and github. Other than views and tests we dont need do to anything else in fb and github as we are using same methods for all providers.

**Comment by SidharthBansal** [comment]
@ryzokuken great to see you back. I hope you had a great trip :smiley:

**Comment by Souravirus** [comment]
Hi @publiclab/soc 
This week plots 2 went through a big change in the form of rails 5. 
But there are some routing problems and some other small errors after the update but we are taking care of it and I along with @jywarren are currently working on these problems.
So, if there is some problem related to any location that you can't get to or any other problem which you think might be due to change to rails 5, then please post it in the issues section so that we can remove it quickly.
Thank you!!

**Comment by SidharthBansal** [comment]
@publiclab/soc dont forget to fill up the evaluation forms this week :1234:

**Comment by Gauravano** [comment]
Hi @publiclab/soc ,
Due to my exams last week, I was not able to perform last week tasks. So, I would be completing them. 

I have created the wiki page for the Draft feature, do check it https://publiclab.org/wiki/draft-feature .

Thanks

P.S. it's good be back after exams. Let's code :muscle: :smiley:

**Comment by jywarren** [comment]
Hi, all! It's so great to see some big feature going into production. Great work! Please work with @icarito to get your environment variables set up, and we'll start to see these activate in production.

I'm a little less available this week as I'm working with a new team member at the New Orleans office - Joe! I hope to introduce him to you all soon! I'm showing him all around the website today actually 🎉 

Folks, keep an eye out for this error -- `There was an error with Danger bot’s Junit parsing: undefined method `value’ for nil:NilClass`

You can restart (close + open) your PR to get past it usually! But any help appreciated in solving it! https://github.com/publiclab/plots2/issues/2824


### CodeClimate

How have the CodeClimate recommendations gone? Are we seeing fewer of them as we get through files? @siaw23 wanted to get Rubocop to auto-fix things, but I'm not familiar with how to set that up. I do think that could help make the CodeClimate recommendations easier to deal with, especially with newcomers. I'm interested to hear your views!

**Comment by jywarren** [comment]
And welcome back @Gauravano -- hope the exams went well!!!! 👍

**Comment by jywarren** [comment]
One more thing - what do we have to do to allow others to manually approve CodeClimate checks? That'd be great...

**Comment by SidharthBansal** [comment]
I think CodeClimate is great. Even though we are going through errors from CodeClimate but later on we will have good code.
It will be great to have one new member Joe at public labs.

**Comment by SidharthBansal** [comment]
@jywarren I want to set up rubocop but after soc evaluations as now we are already doing so many big changes to the plots repo. Can I make an issue for this?

**Comment by jywarren** [comment]
Hi, all - just an update that we have completed all of the mentor evaluations and I see all students have completed their submissions too -- great work!

**Comment by namangupta01** [comment]
Thanks @jywarren, In this week i am working on finding best ways for reply-by-tweet feature and also discussing about how to implement this feature so that we don't end up spamming due to twitter-bots.

**Comment by SidharthBansal** [comment]
Also, please close this issue once you open new issue

**Comment by jywarren** [comment]
Ah, oops! I usually chime in to ask someone... forgot last week! Anyone want to volunteer?

**Comment by ViditChitkara** [comment]
I would like to do this week's checkin!!

**Comment by SidharthBansal** [comment]
Awesome, closing this then

**Comment by ViditChitkara** [comment]
moved to #2840

**Comment by SidharthBansal** [comment]
Awesome, closing this then
